nextcord/abc.py

```diff
@@ -629,6 +629,7 @@ def permissions_for(self, obj: Union[Member, Role], /) -> Permissions:
 
             return base
 
+        # Handle Member case
```

Comment:
What is this comment for?

Reply:
I added it as a way to more easily scope the part of the code I was reading but I will remove it if it seems unnecessary